utils/randomString.js

```diff
@@ -0,0 +1,11 @@
+function randomString(length) {
```

Comment:
Move this function to a file called utils.js
Lets not create a separate file for each utility. It can be exported together from a single file utils.js